src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/model/EfilingDocument.java

```diff
@@ -0,0 +1,341 @@
+package ca.bc.gov.open.jag.efilingcommons.model;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlSchemaType;
+import javax.xml.datatype.XMLGregorianCalendar;
+import java.math.BigDecimal;
+import java.util.List;
+
+public class EfilingDocument {
+    private Boolean amendsAnotherDocumentYn;
+    @XmlSchemaType(name = "dateTime")
+    private XMLGregorianCalendar appearanceDt;
```

Comment:
if we're declaring this many vars, it might help to sort by type and alphabetically for future readability/maintainability. just a thought.